### DISCUSSION TITLE
Where should vkquake.pak go when starting from another directory?

### DISCUSSION THREAD

**Discussion by okkiv** [Novum/vkQuake#373] [Q&A]
Hi @Novum,

is the file vkquake.pak needed to run the game at all?

As far as I understand, the documentation says to unpack all files alongside the id1 directory. So vkquake.pak goes next to vkQuake.AppImage. Being a .pak-file, shouldn't it go into id1?

As a test, I have moved vkQuake.AppImage into .local/bin and tried starting with the -basedir parameter, which did work (unsurprisingly).
I was surprised to find that the game seems to work no matter where I put vkquake.pak, though. I only started a new game and jumped around the difficulty selection portals.

**Comment by Novum** [comment]
It is not strictly required and it should be placed next to the executable
